quickstart/2-browser-parallel-computation/README.md

```diff
@@ -0,0 +1,122 @@
+# 3. Browser-Parallel-Computation
+
+> It's recommended to complete [CLI quickstart](https://fluence.dev/docs/build/get-started) before exploring this example as the example contains concepts build upon deployed service 
+
+In the first section, we explored browser-to-browser messaging using local, i.e. browser-native, services and the Fluence network for message transport. Now we have already deployed a simple service `Adder` for you, and we will try to experiment with this.
+
+The service code:
+
+```rust
+use marine_rs_sdk::marine;
+
+pub fn main() {}
+
+#[marine]
+pub fn add_one(value: u64) -> u64 {
+value + 1
+}
+```
+
+You can deploy it yourself by following [CLI quickstart](https://fluence.dev/docs/build/get-started) guide, replacing `HelloWorld` module with following code and renaming `HelloWorld` service to `Adder` service there.
+
+First, let's have a look at the Aqua file. Navigate to the `aqua/getting_started.aqua` file in your IDE or terminal:
+
+> If you have trouble reading aqua, feel free to refer to [aqua book](https://fluence.dev/docs/aqua-book/introduction) 
+
+> Read carefully through comments, starting with `--`.
+
+```aqua
+-- Import builtin services
+import "@fluencelabs/aqua-lib/builtin.aqua"
+-- Import subnets
+import Subnet, Worker from "@fluencelabs/aqua-lib/subnet.aqua"
+
+-- Deal id of the deployed worker. We need this to resolve subnet 
+const ADDER_DEAL_ID ?= "0x3C85E6765e58Df1e6c09E5CDC0C386bEEBdBa155"
```

Comment:
Let's not advertise hardcoding of deals into aqua. Better migrate these to be fcli projects.

Reply:
Oh crap. I updated aqua but forgot README